README.md

```diff
@@ -8,3 +8,19 @@ Add all of your Research, Meeting Agendas and everything project related.
 ## Contributing
 
 Please see the [contribution guidelines](CONTRIBUTING.md).
+
+
+## What is this repositry used for?
+
+This documentation repo is used to collate all information in regards to the Thoth-Tech Project. 
+
+Each team has their own separate repository and these are supposed to be used for anything related to the team projects. Additionally, if you have documentation elsewhere, we recommend that you begin a small migration process to this repository within a .md format.
+
+Some of the things you might want to include within this repositry includes: 
+
+* Meeting Minutes
+* Research and Findings
+* Onboarding Packages
+* And more - the potential is unlimited.
```

Comment:
Using * as bullet points might be a cause of failing the formatting check

Consider 
- a dash for a list